### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by lftherios**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,26 @@
+radicle-upstream is Copyright © 2019-2020 Monadic GmbH <company@monadic.xyz>
+
+radicle-upstream is licensed under version 3 of the GNU General Public License (GPLv3) with the Radicle Linking Exception. Any contribution to radicle-upstream must be made subject to the developer certificate of origin (DCO).
```
Comment (rudolfs) [inline LICENSE:3]: I'd expect this file to be hard-wrapped.

Comment (rudolfs) [reply LICENSE:3]: Also wonder whether we should not use the same LICENSE file everywhere:
https://github.com/radicle-dev/radicle-bins/pull/53/files \cc @cloudhead

File: LICENSE [hunk 2]
```diff
@@ -619,3 +645,96 @@ Program, unless a warranty or assumption of liability accompanies a
...
+The GNU General Public License does not permit incorporating your program 
+into proprietary programs. If your program is a subroutine library, you 
+may consider it more useful to permit linking proprietary applications with 
+the library. If this is what you want to do, use the GNU Lesser General 
+Public License instead of this License. But first, please read 
+<https://www.gnu.org/licenses/why-not-lgpl.html>.
+
+
+ 
+                           Developer Certificate of Origin
```
Comment (cloudhead) [inline LICENSE:701]:
```suggestion
                   Developer Certificate of Origin
```

File: LICENSE (hunk 1 above)
Comment (lftherios) [reply LICENSE]: fixed. we are following Ange's instructions on this one, so I would be hesitant to diverge.

**Review by xla — DISMISSED** [review]
❕ 👼 ♏️ ✈️

**Review by cloudhead — APPROVED** [review]

**Review by FintanH — APPROVED** [review]
This looks good :+1:

**Review by rudolfs — APPROVED** [review]

**Pull request merged**